OathLab_Twitter/app/src/main/java/net/serkanbal/oathlab_twitter/MainActivity.java

```diff
@@ -0,0 +1,178 @@
+package net.serkanbal.oathlab_twitter;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.google.gson.Gson;
+
+import org.apache.commons.codec.binary.Base64;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.FormBody;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class MainActivity extends AppCompatActivity {
+    public static final String CONSUMER_KEY = "gKOs6rrJhWoH73ZHTjyy33QFH";
+    public static final String CONSUMER_SECRET = "0LF56WARaf02v1KilCfBnpCZAsG2HBAnPAdLQu377FO7BA7vVn";
+    public String mScreenName = "MargotRobbie";
+    List<TwitterObject> mList = new ArrayList<>();
+    EditText mScreenNameEntry;
+    Button mSearch;
```

Comment:
You could make all these variables private, since you won't access them from anywhere outside this class